js/graviex.js

```diff
@@ -0,0 +1,987 @@
+'use strict';
+
+// ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ArgumentsRequired, ExchangeError, InvalidOrder, AuthenticationError } = require ('./base/errors');
+
+// ---------------------------------------------------------------------------
+
+module.exports = class graviex extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'graviex',
+            'name': 'Graviex',
+            'version': 'v3',
+            'countries': [ 'MT', 'RU' ],
+            'rateLimit': 1000,
```

Comment:
> 6000 requests/keypair/5 minutes

means the rateLimit should be lower, in particular 5mins = 5 * 60 = 300 seconds or 300,000 milliseconds, so 300,000 / 6000 = 50ms/req

Reply:
although that's just the private api, for the public api they write "unlimited", but that is certainly not the case...

Reply:
Okay. Fixed in the latest commit https://github.com/ccxt/ccxt/commit/81c8152ac30b15da7aad6a3f117da386d51efd62